src/infra/postgres/client/client.go

```diff
@@ -120,3 +132,24 @@ func (c PostgresClient) RunInTransaction(ctx context.Context, persist func(clien
 
 	return c.db.(*pg.DB).RunInTransaction(ctx, persistFunc)
 }
+
+func (c *PostgresClient) SelectMany(ctx context.Context, model, dst interface{}, condition string, params ...interface{}) error {
+	q := c.db.ModelContext(ctx, model)
+	err := q.Where(condition, params...).Select(dst)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (c *PostgresClient) Ping(ctx context.Context) error {
```

Comment:
I suppose you need this in the migration process? If so, it should not appear here.

Reply:
It will be used for healthcheck -> rename to Healthcheck.

Reply:
check in orchestrate

Reply:
Removed for now, it's handled in another way with the acceptance tests.